blog/_posts/2020-03-18-alumni-interview-jessica.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Alumni Interview with Jessica Greene
+layout: post
+created_at: Weds Mar 18 2020
+permalink: blog/2020-03-18-alumni-interview-jessica
+current: blog
+author: RGSoC Team
+twitter: RailsGirlsSoC
+categories:
+ - alumna-series
+---
+ 
+![](/img/blog/2020/Alumni-Interview-JessicaG.png)
+ 
+> **“My claim to fame is making coffee for the late Christopher Lee!”**
+ 
+Jessica Greene has been living in Berlin for 10 years. After five years in the film & television industry and a stint as a Barista, she took the leap into coding and has since gone on to work as a back-end focused software engineer at Ecosia, the search engine that plants trees.
```

Comment:
```suggestion
Jessica Greene has been living in Berlin for 10 years. After five years in the film & television industry and a stint as a Barista, she took the leap into coding and has since gone on to work as a backend-focused software engineer at Ecosia, the search engine that plants trees.
```